packages/components/src/server-side-render/index.js

```diff
@@ -20,10 +20,11 @@ import { addQueryArgs } from '@wordpress/url';
 import Placeholder from '../placeholder';
 import Spinner from '../spinner';
 
-export function rendererPathWithAttributes( block, attributes = null ) {
+export function rendererPath( block, attributes = null, urlQueryArgs = null ) {
```

Comment:
Minor: I think for this case we may be able to just do urlQueryArgs = {} on the parameter list, and then ...urlQueryArgs inside addQueryArgs.

Reply:
Good call. Done.